cluster/charts/crossplane/README.md

```diff
@@ -30,10 +28,17 @@ helm install --name crossplane --namespace crossplane-system crossplane-alpha/cr
 
 Install with Helm 3:
 
+If your Kubernetes version is lower than 1.15 and you'd like to install Crossplane via Helm 3, you'll need Helm v3.1.0+ that has the flag `--disable-openapi-validation`.
+
 ```console
 kubectl create namespace crossplane-system
 helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+
+# post-1.15 cluster
```

Comment:
should this be 1.15 and after, or 1.15+ cluster? the two headers of pre 1.15 and post 1.15 aren't clear on which bucket 1.15 falls into

Reply:
Just updated to make it more clear